pkg/conversion/converter.go

```diff
@@ -379,7 +414,18 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	if fv, ok := c.funcs[typePair{st, dt}]; ok {
+
+	// Apply default values.
+	if fv, ok := c.defaultingFuncs[st]; ok {
+		if c.Debug != nil {
+			c.Debug.Logf("Applying defaults for '%v'", st)
```

Comment:
Part of the reason it's not glog is that in test cases glog is not properly interspersed with "t.Errorf" messages, which makes debugging tables very very difficult.  It's probably better to leave this, since the only real case for this log is test suites.